docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -0,0 +1,749 @@
+            <td>true</td>
+            <td>If true, allows users to reset their passwords</td>
+        </tr>
+        <tr>
+            <td>`insecureCookies`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, allows insecure cookies.</td>
+        </tr>
+        <tr>
+            <td>`registrationTermsMessage`</td>
+            <td>String</td>
+            <td>
+                `"You must agree to the <a href='http://main.loculus.org/terms'>terms of use</a>."`
+            </td>
+            <td>Message displayed during user registration, typically including terms of service.</td>
+        </tr>
+    </tbody>
+</table>
+
+## Database deployments
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-72'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`runDevelopmentMainDatabase`</td>
+            <td>Boolean</td>
+            <td>true</td>
+            <td>If true, runs a development database within the cluster.</td>
+        </tr>
+        <tr>
+            <td>`runDevelopmentKeycloakDatabase`</td>
+            <td>Boolean</td>
+            <td>true</td>
+            <td>If true, runs a development Keycloak database within the cluster.</td>
+        </tr>
+    </tbody>
+</table>
+
+For production environments, these should typically be set to false. Instead, external managed databases should be used.
+
+## Services
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-48'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`disableWebsite`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables the frontend website deployment.</td>
+        </tr>
+        <tr>
+            <td>`disableBackend`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables the backend API deployment.</td>
+        </tr>
+        <tr>
+            <td>`disablePreprocessing`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables preprocessing pipelines.</td>
+        </tr>
+        <tr>
+            <td>`disableIngest`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables ingestion services.</td>
+        </tr>
+        <tr>
+            <td>`disableEnaSubmission`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables ENA (European Nucleotide Archive) submission service.</td>
+        </tr>
+        <tr>
+            <td>`customWebsiteImage`</td>
+            <td>String</td>
+            <td></td>
+            <td>Optional custom Docker image for the website.</td>
+        </tr>
+    </tbody>
+</table>
+
+## Organism Configuration
```

Comment:
We have a full documentation on this here https://github.com/loculus-project/loculus/blob/main/docs/src/content/docs/for-administrators/setup-with-kubernetes.md#organism-configuration already - would be nice to link this, or otherwise keep things together

Reply:
Ideally avoid duplication, organism config docs have 2 parts I suppose:
- The technical schema (reference style doc)
- Practical considerations (how-to style docs)

The technical should live here, the how-to can remain in a separate space

Reply:
Thanks! I added a link to the reference from the setup-with-kubernetes page. I agree with having different parts/document styles and I think that we are on a good way. The setup-with-kubernetes seems to me like (if we use the terminology of [Diátaxis](https://diataxis.fr/)) a mix of tutorial (as it explains certain concepts) and how-to guide and this document would be a reference.